### DISCUSSION TITLE
Change font in folder view

### DISCUSSION THREAD

**Discussion by v-Nyo** [Taiko2k/Tauon#1050] [Q&A] [answered]
Is it possible to change the font in the folder view?

**Comment by Taiko2k** [comment]
No there's currently no setting to change the Folder View font. Although are you installing Tauon using the AUR? There is currently a bug with Arch Linux where its not using the intended font. Installing using Flatpak uses the intended font. Otherwise maybe a setting for this could be added.

**Reply by v-Nyo** [reply]
Font works fine (flapak), it just really bothers me that my font doesn't match at all with the one in the folder view.

**Reply by v-Nyo** [reply]
and in the artist list, tab, mini, slate and square mode (maybe somewhere else too?)

would appreciate a setting to change those

**Reply by Taiko2k** [reply, marked as answer]
OK Ive committed a change that will allow you to change the condensed font which is used in the folder list and mini mode (though you still cant change those two independently)

**Reply by v-Nyo** [reply]
thanks!
